monkey/monkey_island/cc/server_utils/consts.py

```diff
@@ -11,8 +11,6 @@
     MONKEY_ISLAND_ABS_PATH, "cc", "server_config.json.develop"
 )
 
-DEFAULT_LOGGER_CONFIG_PATH = os.path.join(
-    MONKEY_ISLAND_ABS_PATH, "cc", "island_logger_default_config.json"
-)
+DEFAULT_LOG_LEVEL = "NOTSET"
```

Comment:
Why not `DEFAULT` or `INFO`? Also, if this is only used in `monkey_island.py`, it can probably be moved there.